src/scripts/npm-dependencies.js

```diff
@@ -57,6 +56,18 @@ var fsWrapper = {
           return callback(e);
         }
       }
+    },
+    readFileBytes: async function(path, opts, callback) {
+      if(!window.MESSAGES.sendRpc) { throw new Error("Cannot readFile on the web"); }
+      else {
+        try {
+          const result = await window.MESSAGES.sendRpc('fs', 'readFile', [path]);
```

Comment:
We need to change the extension/these calls to pass `opts` along and use those to control bytes vs utf8 returns